test/RiskManagerV1.test.js

```diff
@@ -393,6 +397,133 @@ describe("RiskManagerV1", () => {
     })
   })
 
+  describe("beginSignerBondsSwapStrategyUpdate", () => {
+    context("when the caller is the owner", () => {
+      let tx
+      beforeEach(async () => {
+        tx = await riskManagerV1
+          .connect(owner)
+          .beginSignerBondsSwapStrategyUpdate(
+            anotherSignerBondsSwapStrategy.address
+          )
+      })
+
+      it("should not update the signerBondsSwapStrategy", async () => {
```

Comment:
Can we make it more human-friendly: `should not update the signer bonds swap strategy`?

Reply:
Fixed in d295b8363bac9bc886cb50c33525c461ac2c3721.